src/platform/assets/components/AssetFilterBar.vue

```diff
@@ -56,17 +56,21 @@ export interface FilterState {
   sortBy: string
 }
 
-const props = defineProps<{
-  assets?: AssetItem[]
-}>()
+const props = withDefaults(
+  defineProps<{
+    assets?: AssetItem[]
+  }>(),
+  {
+    assets: () => []
+  }
+)
```

Comment:
https://vuejs.org/guide/components/props#reactive-props-destructure
```suggestion
const { assets = [] } = defineProps<{
    assets?: AssetItem[]
  }>()
```